### PR TITLE
build: set up samples testing

### PR DESCRIPTION
**Pull request by anguillanneuf**
I created a Dataproc cluster and a GCS bucket for the samples integration testing.

They are loaded via
https://github.com/googleapis/java-pubsublite-spark/blob/5785b66e13db36e33ff21d7ab5be27aadbf9cdd7/.kokoro/build.sh#L80-L85

After this PR is merged, the following (additional) env vars are available for samples testing: 
- `PROJECT_ID`
- `GOOGLE_CLOUD_PROJECT_NUMBER`
- `CLUSTER_NAME`: a Dataproc cluster
- `BUCKET_NAME`: a GCS bucket for storing the connector JAR and the samples JAR
- `CONNECTOR_VERSION`: see script
- `SAMPLE_VERSION`: see script
- `TOPIC_ID`: Lite topic already created with bible.txt published
- `CLOUD_REGION`
- `CLOUD_ZONE`

### CODE REVIEW

File: .kokoro/run_samples_tests.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+# `-o pipefail` sets the exit code to the rightmost comment to exit with a non-zero
+set -eo pipefail
+
+echo "********** MAVEN INFO  ***********"
+mvn -v
+
+# Get the directory of the build script
+scriptDir=$(realpath $(dirname "${BASH_SOURCE[0]}"))
+## cd to the parent directory, i.e. the root of the git repo
+cd ${scriptDir}/..
```
Comment (jiangmichaellll) [inline .kokoro/run_samples_tests.sh:26]: I think it needs to go into the sample folder? mvn from top level won't be able to trigger anything inside sample folder.

**Review by jiangmichaellll — COMMENTED** [review]
If I incorporate in my integration test PR how can I see that this triggers? Does every PR's kokoro run uses the branch's settings?

File: .kokoro/run_samples_tests.sh (hunk 1 above)
Comment (anguillanneuf) [reply .kokoro/run_samples_tests.sh:26]: I missed a line. Added.

File: versions.txt [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # Format:
 # module:released-version:current-version
 
-pubsublite-spark-sql-streaming:0.0.0:0.1.0-SNAPSHOT
+pubsublite-spark-sql-streaming:0.0.0:0.0.1-SNAPSHOT
```
Comment (jiangmichaellll) [inline versions.txt:4]: 0.1.0-SNAPSHOT

**Review by jiangmichaellll — COMMENTED** [review]
LGTM remember to update the version.

**Review by jiangmichaellll — APPROVED** [review]

**Pull request merged**